server/davidia/tests/test_plotserver.py

```diff
@@ -697,7 +782,6 @@ async def test_prepare_data():
         type=MsgType.append_line_data,
         params=[
             {
-                "key": "",
```

Comment:
The pydantic model validation was failing, but that's caught and printed to the error log rather than raising an exception. I've added a pytest fixture that checks the logging.